docs/apache-airflow-providers-cncf-kubernetes/operators.rst

```diff
@@ -73,14 +103,21 @@ and type safety. While we have removed almost all Kubernetes convenience classes
     :start-after: [START howto_operator_k8s_cluster_resources]
     :end-before: [END howto_operator_k8s_cluster_resources]
 
-Difference between ``KubernetesPodOperator`` and Kubernetes object spec
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-The :class:`~airflow.providers.cncf.kubernetes.operators.kubernetes_pod.KubernetesPodOperator` can be considered
-a substitute for a Kubernetes object spec definition that is able
-to be run in the Airflow scheduler in the DAG context. If using the operator, there is no need to create the
-equivalent YAML/JSON object spec for the Pod you would like to run.
-The YAML file can still be provided with the ``pod_template_file`` or even the Pod Spec constructed in Python via
-the ``full_pod_spec`` parameter which requires a Kubernetes ``V1Pod``.
+
+How to use KubernetesPodOperator with YAML file/JSON spec?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * WIP
```

Comment:
Does these `WIP`s mean this PR is still a draft at this moment? If so, let's convert it to draft.

Reply:
Yes. It is more of a draft now - lots more write-ups will be added.